service/store/store_service.go

```diff
@@ -0,0 +1,31 @@
+package store
+
+import (
+	"gin/utils"
+	"github.com/gin-gonic/gin"
+	"github.com/olivere/elastic/v7"
+)
+
+const indexName = "gilad_books"
+
+func fetchStoreDate(es *elastic.Client) (gin.H, error) {
```

Comment:
Let's not have inner functions return gin.H{}. We use it for pretty formatted responses and not for logic.